### PR TITLE
TASK: Allow translation of text field editor placeholder

### PR DESCRIPTION
**Pull request by kitsunet**

### CODE REVIEW

File: TYPO3.Neos/Classes/TYPO3/Neos/Aspects/NodeTypeConfigurationEnrichmentAspect.php [hunk 1]
```diff
@@ -179,6 +179,11 @@ protected function applyInspectorEditorLabels($nodeTypeLabelIdPrefix, $propertyN
                     $propertyConfiguration['ui']['inspector']['editorOptions']['buttonLabel'] = $this->getPropertyConfigurationTranslationId($nodeTypeLabelIdPrefix, $propertyName, 'codeEditor.buttonLabel');
                 }
                 break;
+            case 'TYPO3.Neos/Inspector/Editors/TextFieldEditor';
+                if (isset($propertyConfiguration['ui']['inspector']['editorOptions']) && $this->shouldGenerateLabel($propertyConfiguration['ui']['inspector']['editorOptions'], 'placeholder')) {
```
Comment (johannessteu) [inline TYPO3.Neos/Classes/TYPO3/Neos/Aspects/NodeTypeConfigurationEnrichmentAspect.php]: Should be `:` instead of `;`

**Comment by johannessteu** [comment]
👍  by reading

**Comment by dfeyer** [comment]
Nice one thanks

**Comment by RafaelKa** [comment]
I gues it is not provided by  JS. Also nothing is translated in Inspector, I see only xliff ids.

If i gat it right it should happen in [TYPO3.Neos/Resources/Public/JavaScript/Content/Inspector/Editors/TextFieldEditor.js](https://github.com/neos/neos-development-collection/blob/master/TYPO3.Neos/Resources/Public/JavaScript/Content/Inspector/Editors/TextFieldEditor.js)

**Pull request merged**